[2021-01-11 18:31:14,941] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: MarketVol.apple_price_analyzer 2021-01-12T02:31:05.853405+00:00 [queued]>
[2021-01-11 18:31:14,947] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: MarketVol.apple_price_analyzer 2021-01-12T02:31:05.853405+00:00 [queued]>
[2021-01-11 18:31:14,947] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-11 18:31:14,947] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-01-11 18:31:14,947] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-11 18:31:14,954] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): apple_price_analyzer> on 2021-01-12T02:31:05.853405+00:00
[2021-01-11 18:31:14,956] {standard_task_runner.py:51} INFO - Started process 56698 to run task
[2021-01-11 18:31:14,962] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'MarketVol', 'apple_price_analyzer', '2021-01-12T02:31:05.853405+00:00', '--job-id', '41', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/stock_dags.py', '--cfg-path', '/var/folders/hy/5f75_vt15hb_bfkv9vvnx5sw0000gn/T/tmpkte46e5e']
[2021-01-11 18:31:14,963] {standard_task_runner.py:76} INFO - Job 41: Subtask apple_price_analyzer
[2021-01-11 18:31:14,994] {logging_mixin.py:103} INFO - Running <TaskInstance: MarketVol.apple_price_analyzer 2021-01-12T02:31:05.853405+00:00 [running]> on host Chow-MBP.local
[2021-01-11 18:31:15,018] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=MarketVol
AIRFLOW_CTX_TASK_ID=apple_price_analyzer
AIRFLOW_CTX_EXECUTION_DATE=2021-01-12T02:31:05.853405+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-01-12T02:31:05.853405+00:00
[2021-01-11 18:31:15,018] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/hy/5f75_vt15hb_bfkv9vvnx5sw0000gn/T
[2021-01-11 18:31:15,019] {bash.py:158} INFO - Running command: python /Users/chowli/Documents/Springboard_DE/Projects/mini-projects/Airflow/marketvol/airflow/script/analyze.py --symbol AAPL
[2021-01-11 18:31:15,025] {bash.py:169} INFO - Output:
[2021-01-11 18:31:15,066] {bash.py:173} INFO -   File "/Users/chowli/Documents/Springboard_DE/Projects/mini-projects/Airflow/marketvol/airflow/script/analyze.py", line 8
[2021-01-11 18:31:15,067] {bash.py:173} INFO -     file_path = f'/Users/chowli/Documents/Springboard_DE/Projects/mini-projects/Airflow/marketvol/airflow/tmp/data/{today}')
[2021-01-11 18:31:15,067] {bash.py:173} INFO -                                                                                                                            ^
[2021-01-11 18:31:15,067] {bash.py:173} INFO - SyntaxError: invalid syntax
[2021-01-11 18:31:15,069] {bash.py:177} INFO - Command exited with return code 1
[2021-01-11 18:31:15,075] {taskinstance.py:1396} ERROR - Bash command failed. The command returned a non-zero exit code.
Traceback (most recent call last):
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/Users/chowli/.local/share/virtualenvs/marketvol-WbDfdy4N/lib/python3.7/site-packages/airflow/operators/bash.py", line 180, in execute
    raise AirflowException('Bash command failed. The command returned a non-zero exit code.')
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code.
[2021-01-11 18:31:15,077] {taskinstance.py:1440} INFO - Marking task as FAILED. dag_id=MarketVol, task_id=apple_price_analyzer, execution_date=20210112T023105, start_date=20210112T023114, end_date=20210112T023115
[2021-01-11 18:31:15,105] {local_task_job.py:118} INFO - Task exited with return code 1
